### ISSUE TITLE
App crashing

### ISSUE THREAD

**Issue by dedors** [mooninite/g20s-pro-bt#3]
I tried in on 3 phones, and it always crashed when loading the .dat file.

Solution was to create a folder G20BTSPLUS in the root of the SD, and put the file there.

Either it is because of the foldername, or because I have to access the file not via "Downloads" but via "SD Card".

**Comment by mooninite** [comment]
Yes, the app crashes if you follow their directions. Thanks for the note about the folder. I did not know about it.

My notes on this:

The Android app crashed on my device when following the directions. The file loading step has a menu with three options. Their instructions say to use the file chooser, the first option, and choose the file. After choosing the file the app would crash. I had to copy the firmware file into the app's storage directory and use the 2nd option that looked in its own directory. The third option is a remote server option and there is a firmware file listed, but I do not know what the file is and I would not select it.

**Comment by dedors** [comment]
I used the first option and picked the file myself.

Maybe also some help for someone else: I paired the remote first. Long press OK+Return, then it should appear.
On my fist phone (Nothing Phone (1)), it paired, but could never hold the connection. The remote did not work.
On my second phone (Mi Note 10 with lineage), the remote did work (mouse pointer appears and navigation buttons also work), then I could do the update.

**Comment by dedors** [comment]
I created a[ pull request](https://github.com/mooninite/g20s-pro-bt/pull/4) with detailed steps.

**Comment by DavidSpivey** [comment]
Does anyone have the contact for the Amazon seller where the updated firmware came from? I have a G60s pro with the same annoying OK button issue. I am hoping the company that supplied you with the firmware also can supply me with updated firmware.
